backend/src/routes/v2/tags.ts

```diff
@@ -50,6 +50,7 @@ router.post(
   body("name").exists().trim(),
   body("tagColor").exists().trim(),
   body("slug").exists().trim(),
+  body("checkedSecrets").exists(),
```

Comment:
For tag creation should the checkedSecret be mandatory. Can you tell me why its required?

Reply:
Yes the checked secrets is necessary here @akhilmhdh as it is needed if I want to create tags for multiple secrets. Although I am defaulting to an empty array if no secrets are checked avoid an error.

Reply:
Yah so if your passing empty array then u can keep this optional right